### ISSUE TITLE
mcr.microsoft.com/dotnet/core/sdk:3.1 docker image uses 7.0.0-rc.2 of PowerShell

### ISSUE THREAD

**Issue by ManojNair** [dotnet/dotnet-docker#1727]
Since PowerShell 7.0 is now GA, can we have the .NET Core SDK docker image to include the GA Version

```powershell
❯ docker run --rm -it mcr.microsoft.com/dotnet/core/sdk:3.1 /bin/bash
root@b206a5688965:/# pwsh
PowerShell 7.0.0-rc.2
Copyright (c) Microsoft Corporation. All rights reserved.

https://aka.ms/powershell
Type 'help' to get help.

PS /> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.0.0-rc.2
PSEdition                      Core
GitCommitId                    7.0.0-rc.2
OS                             Linux 4.19.76-linuxkit #1 SMP Thu Oct 17 19:31:58 UTC 2019
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

Installing PowerShell as a dotnet global tool does install the PowerShell 7 GA version.

**Comment by mthalman** [comment]
Yes, it will be available in the next servicing release for the 3.1 images.  It's actually available right now in the nightly images: `mcr.microsoft.com/dotnet/core-nightly/sdk:3.1`

**Comment by ManojNair** [comment]
Thank you @mthalman
